### PR TITLE
[s] Fixes 2 bugs/exploits

### PR DESCRIPTION
**Pull request by Time-Green**
Fixes being able to pull an infinite amount of replicant alloy out of a cache

Fixes being able to repair partially broken screens by screwdrivering it. Screwdrivering a partially broken screen will now break it, cuz fuck you for exploiting my feature.

:cl:
fix: Fixes infinite replicant alloy bug
fix: Fixes instant repairing screens with screwdriver
/:cl:

### CODE REVIEW

File: code/game/gamemodes/clock_cult/clock_structures.dm [hunk 1]
```diff
@@ -221,10 +221,14 @@
 	if(!clockwork_component_cache["replicant_alloy"])
 		user << "<span class='warning'>There is no Replicant Alloy in the global component cache!</span>"
 		return 0
-	var/obj/item/clockwork/component/replicant_alloy/A = new(get_turf(src))
-	user.visible_message("<span class='notice'>[user] withdraws [A] from [src].</span>", "<span class='notice'>You withdraw [A] from [src].</span>")
-	user.put_in_hands(A)
-	return 1
+	if(clockwork_component_cache["replicant_alloy"])
```
Comment (Cruix) [inline code/game/gamemodes/clock_cult/clock_structures.dm]: The block above this already checks if there is any alloy, the only thing you need to add is the line below this one.

Comment (Time-Green) [reply code/game/gamemodes/clock_cult/clock_structures.dm]: Woops, thanks

**Comment by Mrcreepssss** [comment]
how can we fix these screens now besides smashing it and then welding? i mean partially borked windows can be fixed with welder.

**Comment by Time-Green** [comment]
You can just weld to fix.
This only makes the screen shatter if the computer has damage and it gets screwdrivered.
People were screwdrivering almost broken computers to insta-heal them. Screwdrivering damaged screens shatters them, just like the completely broken ones

**Pull request merged**